api/src/test/java/uk/gov/dhsc/htbhf/claimant/entity/PaymentCycleTest.java

```diff
@@ -30,4 +35,35 @@ void shouldFailToValidatePaymentCycleWithNoClaim() {
         //Then
         assertThat(violations).hasSingleConstraintViolation("must not be null", "claim");
     }
+
+    @Test
+    void shouldApplyVoucherEntitlement() {
+        //Given
+        PaymentCycle paymentCycle = aValidPaymentCycleBuilder()
+                .voucherEntitlement(null)
+                .totalEntitlementAmountInPence(null)
+                .totalVouchers(null)
+                .build();
+        PaymentCycleVoucherEntitlement entitlement = aValidPaymentCycleVoucherEntitlement();
+        Assertions.assertThat(entitlement.getTotalVoucherEntitlement()).isGreaterThan(0);
```

Comment:
Can we use a static import here

Reply:
Sadly not. There's already a static import for a different assertThat.

Reply:
Should we switch the assertJ to be a static import seeing as it's used more than the constraint violation one?

Reply:
this is the only test (of 3) that uses this version of assertThat - the constraint violation is actually more common in this class), I think